server/adaptors/integrations/__data__/repository/aws_vpc_flow/data/sample.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "@timestamp": "2023-07-17T08:14:05.000Z",
+    "body": "2 530679726453 eni-0e250409d410e1290 162.142.125.177 10.0.0.200 38471 12313 6 1 44 1674898496 1674898507 ACCEPT OK",
+    "event": {
+      "result": "ACCEPT",
+      "name": "flow_log",
+      "domain": "vpc.flow_log"
+    },
+    "attributes": {
+      "data_stream": {
+        "dataset": "vpc.flow_log",
+        "namespace": "production",
+        "type": "logs"
+      }
+    },
+    "cloud": {
+      "provider": "aws",
+      "account": {
+        "id": "443273586136"
+      },
+      "platform": "aws_vpc",
+    },
+    "aws": {
+      "s3": {
```

Comment:
`aws.s3` is not defined in any of the provided mapping files, so this data would just be dropped if a user tried to load this sample.

Reply:
ok, updated the s3-1.0.0 mapping for aws.s3